src/Provider/FileProvider.php

```diff
@@ -210,10 +210,18 @@ public function getHelperProperties(MediaInterface $media, $format, $options = [
 
     public function generatePrivateUrl(MediaInterface $media, $format)
     {
-        if (MediaProviderInterface::FORMAT_REFERENCE === $format) {
+        if (self::FORMAT_REFERENCE === $format) {
             return $this->getReferenceImage($media);
         }
 
+        // throw new \InvalidArgumentException(sprintf(
```

Comment:
Should we trigger a deprecation then ?

Reply:
I thought about this, but it will be unfixable.